cpp/ql/src/Security/CWE/CWE-020/SafeExternalAPIFunction.qll

```diff
@@ -13,6 +13,7 @@ abstract class SafeExternalAPIFunction extends Function { }
 /** The default set of "safe" external APIs. */
 private class DefaultSafeExternalAPIFunction extends SafeExternalAPIFunction {
   DefaultSafeExternalAPIFunction() {
+    // implementation note: this should be based on the properties of public interfaces, rather than accessing implementation classes directly.  When we've done that, the three classes referenced here should be made fully private.
```

Comment:
@MathiasVP this comment is pretty much directed at you.  It's not urgent, but it would be cleaner all round and will probably produce better results if we figure out the right part of the models _interface_ to use here instead.

Reply:
Thanks for the heads up. I'll create an issue for this.

Reply:
Done: https://github.com/github/codeql-c-analysis-team/issues/189

Reply:
Thanks.